### DISCUSSION TITLE
Packagemanifest format will be deprecated

### DISCUSSION THREAD

**Discussion by J0zi** [k8s-operatorhub/community-operators#2464] [Announcements]
Dear OpenShift Operator Community,

`Packagemanifest` format will no longer be supported from July 2023. The only supported format will be [`bundle`](https://docs.openshift.com/container-platform/4.12/operators/understanding/olm-packaging-format.html#olm-bundle-format_olm-packaging-format) format. You can migrate using `operator-sdk` binary, please follow the [instructions](https://docs.openshift.com/container-platform/4.12/operators/operator_sdk/osdk-pkgman-to-bundle.html).

Anyone who has not migrated by June 30th will be auto-converted to bundle format in a one-off script we are running. More information about the script later. Please stay tuned.

Kind Regards,
Community operators team

mentions:
@2uasimojo, @3deep5me, @3scale, @AdheipSingh, @Aisuko, @Allda, @AppdAgents, @ArangoGutierrez, @ArbelNathan, @BaruchBilanski, @DanArlowski, @DanielePalaia, @EnterpriseDB, @FlorentinDUBOIS, @FushuWang, @HammerMeetNail, @J0zi, @Jainbrt, @JohnStarich, @JohnStrunk, @Jooho, @Julien-Ben, @KohlsTechnology, @Kong, @KoppulaRajender, @NissesSenap, @Ompragash, @PabloB94, @RafalSkolasinski, @RobotSail, @SergK, @Sosivio, @Sosivio-Bot, @Sugar-pack, @ValClarkson, @VineethReddy02, @abutcher, @ack-bot, @acornett21, @adambkaplan, @aimichelle, @akashkumar1982, @akhil-rane, @alien-ike, @andrewlecuyer, @arnavs7, @ashashiv, @ashishshinde, @baijum, @barkbay, @benjaminjb, @bhagyashree-sarawate, @burmanm, @caigy, @camilamacedo86, @cap1984, @carlosedp, @carlosthe19916, @celenechang, @chanwit, @chatton, @chen-keinan, @christophd, @cloudant, @cnbu-jenkins, @cniackz, @cnp-autobot, @czunker, @dalejrodriguez, @danielpacak, @datianshi, @davidebianchi, @davidesalerno, @devOpsHelm, @devopstales, @didierofrivia, @dkwon17, @dlom, @dmesser, @droopy4096, @dt-team-kubernetes, @ebaron, @eguzki, @ejweber, @erusso7, @esara, @eumel8, @external-secrets, @f41gh7, @fabritsius, @fao89, @fbladilo, @fbm3307, @fedepaol, @freeznet, @frzifus, @gauravshankarcan, @ggeorgiev-gitlab, @gl-distribution-oc, @gsreg, @guicassolato, @gusfcarvalho, @haibinxie, @hasancelik, @hco-bot, @helderjs, @hj-johannes-lee, @houshengbo, @iam-veeramalla, @iamabhishek-dubey, @iblancasa, @igor-karpukhin, @imilchev, @infinispan-qe-bot, @instana, @instanacd, @ishitasequeira, @isimluk, @jacky-xbb, @jaymebrd, @jbusche, @jensfr, @jhrozek, @jkremser, @jlong49, @jlopes-randoli, @jmazzitelli, @jmesnil, @joelddiaz, @john-westcott-iv, @jotak, @jstuever, @k8gb-io, @k8s-operatorhub, @keiran-ibm, @khewonc, @ktenzer, @kube-green, @kubero-dev, @lbroudoux, @lleshchi, @loganmc10, @lsierant, @maistra, @matancarmeli7, @maxcao13, @mbmcmullen27, @meik99, @mgoerens, @mgubaidullin, @microcks, @mitchnielsen, @mmontes11, @mms-build-account, @mms-gianni, @mongodb-forks, @moolen, @mporrato, @mshitrit, @mtyazici, @mvala, @mvalarh, @mzegla, @naemono, @ndmspc, @nikhil-thomas, @nmarukovich, @npecka, @ohmyglb, @oriyarde, @oscerd, @paulofrazao, @pavolloffay, @pdhanoya-splunk, @periklis, @pmacik, @pmalek, @portworx, @ppatierno, @project-openubl, @pure-jliao, @pursultani, @qbarrand, @quay, @quay-devel, @rabbitmq, @rafal-lal, @rasapala, @rawagner, @razo7, @reederc42, @rensyct, @rodrigovalin, @rooftopcellist, @routernetes, @rubenvp8510, @rurikudo, @rvansa, @ryanemerson, @sadlerap, @saschagrunert, @scholzj, @scottexton, @sematext, @shruthihub, @sm43, @snyk, @snyksec, @spilchen, @spolti, @streamnative, @streamnativebot, @sud82, @suhanime, @sunya-ch, @sxd, @tahmmee, @team-k8s-bot, @tedhtchang, @teoincontatto, @thbkrkr, @tjmoore4, @tkohn, @tmihalac, @tolusha, @tthvo, @turbonomic, @vaibhavjainwiz, @vanessarakoczy, @vboulineau, @vivekr-splunk, @vmuzikar, @wallrj, @welshDoug, @windup, @windupgithubbot1, @wmellouli, @wojciechka, @yossig-aquasec, @yuriolisa, @zach-robinson, @zfrhv, @zhiweiyin318, @zroubalik

**Comment by acornett21** [comment]
@J0zi Did you tag people that have raise PR's recently? Or are all the people tagged ones that you all see with packagemanifest repos?
